Assets/Scripts/Genetics/rng_functions.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class rng_functions : MonoBehaviour
+{ 
+    /** We asume that features are designed as a range of integers */
+
+    /** Mutate means getting a different feature value than the parents */
+    int mutateFeature(int targetFeature, int featureRange)
+    {
+      direction = Random.Range(0, 2)-1;
+      ammount = Random.Range(0, 2);
+      return targetFeature + (direction * ammount % featureRange);
+    }
+
+    /** Inherit the same feature value as one of the parents or mutate */
+    int inheritFeature(int featureA, int featureB, int featureRange, int mutationProvability = 0)
+    {
+      provA = (100 - mutationProvability)/2;
```

Comment:
change provA and provB names and probability (with b)

Reply:
indeed